_shared/project/Makefile

```diff
@@ -219,34 +195,18 @@ docker-run:
 .PHONY: clean
 $(call help,make clean,"delete temporary files etc")
 clean:
-	@rm -rf build dist .tox
+	@rm -rf build dist .tox .coverage coverage .eslintcache node_modules supervisord.log supervisord.pid yarn-error.log
 	@find . -path '*/__pycache__*' -delete
 	@find . -path '*.egg-info*' -delete
 {% if include_exists("make_clean") %}
 	{{- include("make_clean") -}}
 {% endif %}
 
-{% if cookiecutter.get("frontend") == "yes" %}
-dev: build/manifest.json
-devdata: build/manifest.json
-shell: build/manifest.json
-functests: build/manifest.json
-build/manifest.json:
-	@yarn build
-
-frontend-format: node_modules/.uptodate
-frontend-checkformatting: node_modules/.uptodate
-frontend-lint: node_modules/.uptodate
-frontend-test: node_modules/.uptodate
-build/manifest.json: node_modules/.uptodate
-node_modules/.uptodate: package.json yarn.lock
-	@echo installing javascript dependencies
-	@yarn install
-	@touch $@
-
-{% endif %}
 .PHONY: python
 python:
 	@bin/make_python
 
 -include {{ cookiecutter.package_name }}.mk
+{% if cookiecutter.get("frontend") == "yes" %}
+-include frontend.mk
+{% endif %}
```

Comment:
`-include frontend.mk` will import `frontend.mk` _if_ it exists or pass silently otherwise. And the `-include frontend.mk` line will only be added to the project's `Makefile` if the project has the `frontend` option enabled in `cookiecutter.json`.